requirements/base.txt

```diff
@@ -49,7 +49,7 @@ django-ipware==2.0.2
 geoip2==2.8.0
 django-silk==2.0.0
 django-extensions==2.2.1
-ipdb==0.11
+ipdb==0.13.9
```

Comment:
Was this bumped intentionally? Just want to make sure it won't cause any issues since it wasn't mentioned in the PR description

Reply:
this was brought in from another pr => https://github.com/gitcoinco/web/pull/9443